src/FSharp.Formatting.Common/YaafFSharpScripting.fs

```diff
@@ -201,6 +201,7 @@ module internal CompilerServiceExtensions =
             [| //yield "--debug:full"
                //yield "--define:DEBUG"
                //yield "--optimize-"
+               yield "--langversion:preview"
```

Comment:
Is there any reason why users might need to opt out of that? Can't think of any myself.

Reply:
Possibly. You could imagine an fssnip showing off a feature in preview where we decide to change the syntax. That snippet would then break when fssnip updates to a later FCS. But in general I think the ability to show preview features outweighs the small likelihood that we would do something to a preview feature that led to big breaking changes).

Other than that, there's no real mechanical downside. The compiler used to generate data for formatting already has the preview codepaths, they're just not activated unless you flip this flag.